### DISCUSSION TITLE
Desing for Main Page

### DISCUSSION THREAD

**Discussion by PBC003** [Arquisoft/wiq_es04a#201] [General]
I have several design ideas for the main page. The photos below are of prototypes, that is, it is not the final version but they have been moderately implemented.

The idea is to vote among all which one we like best in order to finish developing that design.

Some of the considerations that remain to be developed are modifying the shape of the logo and resizing images and components.

There are mainly 3 designs, but each one has its variants:

1. Initial design of deliverable II:

       - Variant 1: Leave it as it was

![Screenshot_6](https://github.com/Arquisoft/wiq_es04a/assets/132336640/7c77e2f5-1046-41c5-9a95-59b023d09a48)

       - Variant 2: Only change the background and change it to that of particles

![Screenshot_5](https://github.com/Arquisoft/wiq_es04a/assets/132336640/fa4cc46f-0e5b-4040-95a2-128f908c6324)

2. Alternative design of the main page:

       - Variant 1: Without play button

![Screenshot_3](https://github.com/Arquisoft/wiq_es04a/assets/132336640/958ad6a6-8db7-45b1-a649-39608bde5b2c)
![Screenshot_4](https://github.com/Arquisoft/wiq_es04a/assets/132336640/a8a96aef-3058-4e8e-b5f1-6329037746ca)

       - Variant 2: With play button that takes you to login

![Screenshot_1](https://github.com/Arquisoft/wiq_es04a/assets/132336640/09e2e314-d8e4-4eea-8689-25a03eab4261)
![Screenshot_2](https://github.com/Arquisoft/wiq_es04a/assets/132336640/f4d727f7-44bf-4034-a0fa-218eb5f45186)

3. Simple design, just a background

![Screenshot_7](https://github.com/Arquisoft/wiq_es04a/assets/132336640/048d5666-119e-4ded-9e56-848d4493f669)

4. Minimalist design

       - Variant 1: Logo only
![Screenshot_8](https://github.com/Arquisoft/wiq_es04a/assets/132336640/09266bcd-68de-4c99-97bf-3173835891c7)

       - Variant 2: Logo and Play
![Screenshot_9](https://github.com/Arquisoft/wiq_es04a/assets/132336640/11938e4b-dfa8-4b5a-a273-ded6158c3ad7)

5. Half Page Design

![Screenshot_10](https://github.com/Arquisoft/wiq_es04a/assets/132336640/fc39d101-84a4-4e60-b41a-013b9c17021a)
![Screenshot_11](https://github.com/Arquisoft/wiq_es04a/assets/132336640/e41e6942-7b4c-4c07-aa2e-fcd1e9bca6ba)

**Comment by uo288543** [comment]
I vote: 
![image](https://github.com/Arquisoft/wiq_es04a/assets/145331195/148912c1-2e1e-42c3-a5fc-ca432408f35e)

**Comment by alagoconde** [comment]
I prefer the following one because i like the logo in the middle and the button to play, in my opinion the photo is redundant:
![Screenshot_9](https://github.com/Arquisoft/wiq_es04a/assets/132336640/11938e4b-dfa8-4b5a-a273-ded6158c3ad7)

**Comment by PabloGOP** [comment]
I go with @uo288245. Besides, If we ll have an about us page it is not necessary the text.

However, I d choose not to change the background and leave Don Jordi as it is.

**Comment by donatomartin** [comment]
Wuq, Gaber, Jabe 

![BasedBasedDepartmentGIF](https://github.com/Arquisoft/wiq_es04a/assets/92989546/b55dd24e-20b0-43d8-b354-b061eba5e1f9)

**Comment by and1na** [comment]
I really love the second variant, the one with the button on the left. Moreover, I also like the one proposed by @uo288245, so I’d be happy with either option.:)

**Comment by uo288347** [comment]
I like @PabloGOP 's idea but I like @uo288543 option too.

**Comment by samuel-bustamante** [comment]
I really like the idea from @PabloGOP, although I think all the designs are great.
